gpt_prompter.py

```diff
@@ -0,0 +1,58 @@
+# secret key is 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
+
+import openai
+import asyncio
+import file_writer
+
+from parser import Parser, get_text_from_slide
+
+openai.api_key = "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"
+
+
+def generate_summary(parser: Parser):
```

Comment:
You code doesnt seem to actually run asynchronously. Use logs to validate that, and fix it